### PR TITLE
CB-364 - Ldap v4 related integration test

### PR DESCRIPTION
**Pull request by attilapalfi92**

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ldap/LdapConfig.java [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
 import com.sequenceiq.it.cloudbreak.newway.action.Action;
+import com.sequenceiq.it.cloudbreak.newway.action.ldap.LdapConfigDeleteAction;
 import com.sequenceiq.it.cloudbreak.newway.action.ldap.LdapConfigPostAction;
 import com.sequenceiq.it.cloudbreak.newway.context.TestContext;
 import com.sequenceiq.it.cloudbreak.newway.v4.LdapConfigV4Action;
 
 @Prototype
 public class LdapConfig extends LdapConfigEntity {
-    private static final String LDAPCONFIG = "LdapCONFIG";
+
```
Comment (topolyai5) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ldap/LdapConfig.java]: pls, change to the new conception (TestDto/TestAction..)

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ldap/LdapConfig.java [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+    private static final String LDAPCONFIG = "LDAP_CONFIG";
 
     private LdapConfig() {
```
Comment (topolyai5) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ldap/LdapConfig.java]: remove old Ldap tests and related codes

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ldap/LdapConfig.java (hunk 1 above)
Comment (attilapalfi92) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ldap/LdapConfig.java]: I cannot use the new naming convention till we do not agree on refactoring the whole int test code base. I won't rename LdapConfig to LdapConfigTestService and make it extend the LdapConfigTestDto. And if it does no extend the Dto, then the tests won't start, so there is need for a bigger rework here.

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ldap/LdapConfig.java (hunk 2 above)
Comment (attilapalfi92) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/entity/ldap/LdapConfig.java]: Can I delete every old tests that use the old IntegrationTestContext and this Ldap?

**Pull request merged**